doc/man3/EVP_PKEY_keygen.pod

```diff
@@ -101,9 +101,10 @@ a B<size_t> parameter must be given to specify the size of the RSA key.
 If I<type> is C<EC>,
 a string parameter must be given to specify the name of the EC curve.
 If I<type> is C<X25519>, C<X448>, C<ED25519>, C<ED448>, or C<SM2>
-no further parameter is needed. Other key types may be possible if they are
-supplied by the loaded providers. EVP_PKEY_Q_keygen() may be usable with such
-key types as long as they do not require further parameters.
+no further parameter is needed. With OpenSSL versions > 3.4, other key
```

Comment:
Is it including the 3.4 version or not?

Reply:
I would then say `Starting from OpenSSL version 3.5, ....`

Reply:
I used the chosen wording to also include "master", i.e., way before 3.5 gets released.

Reply:
Good to know. So changed in latest commit.